### ISSUE TITLE
UI: Current problems with page display

### ISSUE THREAD

**Issue by SharonCocker** [PTFS-Europe/rebus-list-2#106]
Hi,  

I'm currently in the public view and find that pages are displaying rather oddly, in particular icons.  The same problem still occurs when logged in to admin and this happens in both Chrome and Firefox.  Please see the screenshots below. 

Thanks,
Heather

![image](https://cloud.githubusercontent.com/assets/12249067/13851180/11ad59e6-ec55-11e5-8583-f349cffc6106.png)

![image](https://cloud.githubusercontent.com/assets/12249067/13851199/2048fd7a-ec55-11e5-9bba-587aefa93063.png)

![image](https://cloud.githubusercontent.com/assets/12249067/13851242/4781ac3e-ec55-11e5-8809-eaf95818c01b.png)

**Comment by AndrewIsh** [comment]
Hi Heather

It looks like a font file isn't loading properly. Can you refresh the page and see if that fixes it?

Many thanks
Andrew

**Comment by SharonCocker** [comment]
Hi Andrew,

I tried refreshing the page on Friday, but it didn't seem to fix it.  However, the display is okay today, but instead my lists have disappeared from view when I'm not logged in (ie public view).   I've logged in and my lists are there and published.   I'd planned to delete them and create a better structure so took the opportunity to do that this morning and then see if my new lists were visible, but they still don't seem to appear.   I am doing something wrong?   

Thanks,
Heather

Public browse view of Sheffield's lists (when not logged in):
![image](https://cloud.githubusercontent.com/assets/12249067/13918587/51198a60-ef60-11e5-8fdf-3b15687d12b8.png)

Admin browse view when logged in:
![image](https://cloud.githubusercontent.com/assets/12249067/13918686/cd88ae82-ef60-11e5-81ee-11eb814354c1.png)
